### PR TITLE
Sync with upstream (3.0.12-12.1)

### PR DESCRIPTION
**Pull request by Wescoeur**
Koji build (v8.3-incoming, scratch) : http://koji.xcp-ng.org/taskinfo?taskID=49949

### CODE REVIEW

File: SPECS/sm.spec [hunk 1]
```diff
@@ -360,6 +375,37 @@ The package contains a fake key lookup plugin for system tests
...
+- - CP-46807: reduce logs from scheduler set errors
+- * Mon Jan 15 2024 Mark Syms <mark.syms@citrix.com> - 3.0.12-7
+- - CA-386281 CIFS username can be omitted in ISO SR
+- - CA-386479: ensure we login to all iSCSI Target Portal Groups
+- - CP-46863 Dump Multipath Status from Storage Manager
+- * Wed Nov 22 2023 Tim Smith <tim.smith@citrix.com> - 3.0.12-6
+- - Backport fix for CA-384030 from upstream
+- - CA-377454: ensure iscsiadm lock file is present
+- - CP-45927: Change the pathselector used on Equalogic 100E-00
+- - CA-384783 Probe for NFS4 when rpcinfo does not include it
```
Comment (stormi) [inline SPECS/sm.spec:406]: We already have this quoted changelog entry below.

File: SOURCES/0026-fix-NFSSR-ensure-we-can-attach-SR-during-attach_from.patch [hunk 2]
```diff
@@ -1,4 +1,4 @@
-From c7340e7db9fafbcd106d437212857ff79fdcc9ac Mon Sep 17 00:00:00 2001
+From 681198c41cdc11445d257c181262886676b914a4 Mon Sep 17 00:00:00 2001
```
[the remark this answers is not in the document]
Comment (Wescoeur) [reply SOURCES/0026-fix-NFSSR-ensure-we-can-attach-SR-during-attach_from.patch:1]: I just checked the git history, indeed it was not in the list of applied patches... Thanks!

**Comment by Wescoeur** [comment]
Koji build (v8.3-incoming, scratch) : http://koji.xcp-ng.org/taskinfo?taskID=50028

I added the missing patch to repair the CI pipeline.

**Review by stormi — APPROVED** [review]

**Comment by stormi** [comment]
Delaying build for after we've rebuilt the build dependencies.

**Pull request merged**